roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -141,7 +143,9 @@
     dest: "{{ tempdir }}/templates/oauth-client.yml"
   vars:
     kibana_hostname: "{{ openshift_logging_kibana_hostname }}"
+    kibana_ops_hostname: "{{ openshift_logging_kibana_ops_hostname }}"
```

Comment:
Ideally we would have a module for oc_oauthclient which would be able to figure out that the oauthclient already exists, and just add the kibana-ops hostname to it.

Next, I guess we need to fix oc_obj - either it isn't detecting that the kibana-proxy oauthclient already exists, or it thinks it needs to replace the entire thing.